packager/media/base/pssh_generator_unittest.cc

```diff
@@ -146,23 +201,72 @@ TEST(PsshGeneratorTest, GeneratePlayReadyPsshFromKeyIds) {
   const std::vector<std::vector<uint8_t>> kTestKeyIds = {GetTestKeyId1(),
                                                          GetTestKeyId2()};
   std::unique_ptr<PlayReadyPsshGenerator> playready_pssh_generator(
-      new PlayReadyPsshGenerator());
+      new PlayReadyPsshGenerator(FOURCC_NULL));
```

Comment:
I used `NULL`, because `GeneratePsshFromKeyIds` that is being tested is not implemented, so in any case the test passes, since it expects `GeneratePsshFromKeyIds` to fail. It doesn't matter which value it will be, so I thought that `NULL` is the least weird. But okay, changed it to `cenc`.